src/modules/baseCompare/baseCompare.js

```diff
@@ -0,0 +1,110 @@
+import GitHubAPI from 'github'
+import config from 'config'
+import ScreenshotBucket from 'server/models/ScreenshotBucket'
+
+async function fallbackToMaster(repository) {
+  const buckets = await ScreenshotBucket.query()
+    .where({
+      branch: 'master',
+      repositoryId: repository.id,
+    })
+    .orderBy('id', 'desc')
+  return buckets[0] || null
+}
+
+export default async function baseCompare(options) {
```

Comment:
The reference `options` is useless and the signature of the method is less explicit, you have to read the content of the method to know how to call it.